### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by mohitdhatrak**
Change log: Added product listing page

It is responsive too, screenshots-

image

image

image

### CODE REVIEW

**Review by Rishi05051997 — COMMENTED** [review]
In some places you used for specifically for margin & padding em & in some places you used rem is there any reason? other things are Ok for me

**Comment by mohitdhatrak** [comment]
> In some places you used for specifically for margin & padding em & in some places you used rem is there any reason? other things are Ok for me

I used it as em mostly where i want it to scale according to font size, mostly i wont change size so it really doesn't matter if its em or rem

File: Styles/product.css [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    cursor: pointer;
+    background-color: transparent;
+    border: none;
+    text-decoration: underline;
+    text-transform: uppercase;
+    font-weight: 500;
+}
+
+.filter-partition {
+    border: 0.5px solid #000;
```
Comment (sharmayatendra) [inline Styles/product.css:76]: you can use variable for color

**Review by sharmayatendra — APPROVED** [review]
overall it's good. one suggestion is that you can put all your media queries at the end.

File: Styles/product.css (hunk 1 above)
Comment (mohitdhatrak) [reply Styles/product.css:76]: actually i have asked Vikram in doubt thread regarding this issue already. Will update as per his response.

**Comment by mohitdhatrak** [comment]
> overall it's good. one suggestion is that you can put all your media queries at the end.

I think for finding the CSS later and making changes, it would be better to have media queries below respective element as it will help me quickly find it if i need to change something. If all are at the end then it gets confusing. Someone recommended this to me. I think it's a personal preference because as such it doesn't affect readability directly.

**Pull request merged**